scripts/userconf.sh

```diff
@@ -107,12 +107,16 @@ fi
 ## add these to the global environment so they are available to the RStudio user
 echo "HTTR_LOCALHOST=$HTTR_LOCALHOST" >> ${R_HOME}/etc/Renviron.site
 echo "HTTR_PORT=$HTTR_PORT" >> ${R_HOME}/etc/Renviron.site
+
+
+exclude_vars="HOME PASSWORD"
 ## Set our dynamic variables in Renviron.site to be reflected by RStudio
 for file in /var/run/s6/container_environment/*
 do
-  if [ "${file##*/}" != "HOME" ]; then
-    echo "${file##*/}=$(cat $file)" >> ${R_HOME}/etc/Renviron.site
-  fi
+  ## Remove variable if it already exists, to avoid repeated declaration on container-restart:
+  sed -i "/$file/d" ${R_HOME}/etc/Renviron.site
```

Comment:
It seems to work if we change it like follows?

```shell
sed -i "/^${file##*/}/d" ${R_HOME}/etc/Renviron.site
```

Reply:
right, thanks for testing. updated.